tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -17,6 +17,7 @@ IncludeByKey.DEEPBUFFER_FW_DMA_MS{
 Object.Dai {
 	HDA.0 {
 		name $HDA_ANALOG_DAI_NAME
+		dai_index 0
```

Comment:
The commit leaves me a bit hanging on why these changes are made. The "This is in preparation" is information, but at least for me, this still opens questions. What it is the "instance" id, why we are adding one, how should I set it, etc.

Reply:
@kv2019i @lyakh hopefully the commit messages give you more context now.

Reply:
Thanks @ranj063 the expanded description is very helpful.